### PR TITLE
[412] - handle metadata changes for notifications endpoint

### PR DESCRIPTION
**Pull request by DaDer**
Trello: https://trello.com/c/sLWR8utJ/412-5-partner-app-v2-ios-partner-customized-notifications-tab-icon-add-badge-with-counter

### CODE REVIEW

**Review by coolbnjmn — APPROVED** [review]
LGTM 👍

File: CreatubblesAPIClient/Sources/Mapper/NotificationMetadataMapper.swift [hunk 1]
```diff
@@ -27,11 +27,15 @@ import UIKit
 import ObjectMapper
 
 class NotificationMetadataMapper: Mappable {
+    var totalNewCount: Int?
```
Comment (sebastianewak) [inline CreatubblesAPIClient/Sources/Mapper/NotificationMetadataMapper.swift]: It'd be helpful to add a comment stating what each field means e.g. `new` vs `unread`. Is `unread` a notification that has been opened e.g. go to details or one that hasn't been displayed on a list?

**Review by sebastianewak — APPROVED** [review]
Code: +1
Func: n\a

Looks good, just one suggestion regarding adding a comment explaining the meaning of the new fields.

**Review by JakubJankowski — APPROVED** [review]
+1 to Sebastian's comment, otherwise LGTM 👍

**Comment by DaDer** [comment]
It's odd but the CircleCI is not updating its state. The build passed so I am going to merge it

**Pull request merged**